web_src/js/bootstrap.js

```diff
@@ -7,9 +7,18 @@ import {joinPaths} from './utils.js';
 // This file must be imported before any lazy-loading is being attempted.
 __webpack_public_path__ = joinPaths(window?.config?.assetUrlPrefix ?? '/', '/');
 
+const ignoreErrors = [
+  /vs\.editor\.nullLanguage/, // https://github.com/microsoft/monaco-editor/issues/2962
+];
+
```

Comment:
This block could be put into `showGlobalErrorMessage`.

Because usually (most time) there is no error, then no need to prepare this array for every page loading, slightly saves browser CPU

Reply:
adjusted